lib/grape-swagger/doc_methods.rb

```diff
@@ -491,7 +497,7 @@ def setup(options)
         end
 
         ops = visible_ops.group_by do |route|
-          @@documentation_class.parse_path(route.route_path, api_version)
+          @@documentation_class.parse_path(route.route_path, api_version, hide_module_from_path)
```

Comment:
@dblock So, using class variables precludes the ability to define multiple apps with different swagger doc options as the class variables are defined internally on the DocMethods module(the one and only) rather than the anonymous documentation class that is created in `add_swagger_documentation` implementation. My tests create two apps with different options but fail with the class variable approach because it gets clobbered whatever is the value of the 2nd app. This is obviously a bigger issue than my one little new option but that happens to be caught by the way I wrote my test. What are your thoughts?